external/webserver/http_client.c

```diff
@@ -871,7 +871,7 @@ void http_handle_file(struct http_client_t *client, int method, const char *url,
 	}
 }
 
-static int prepare_chunk_body(char *buf, int len, const char *body, bool is_last_msg)
+static int prepare_chunk_body(char *buf, int len, const char *body, int body_len, bool is_last_msg)
```

Comment:
regarding len & body_len, size_t (unsigned int) is better option.
Of course sometimes some of defect tool can detect it as a issue.

Reply:
done